src/components/User/MyPageCount.tsx

```diff
@@ -0,0 +1,17 @@
+import { Flex, Text } from '@chakra-ui/react';
+
+type MyPageCountProps = {
+  name: string;
+  value: number | undefined;
```

Comment:
p5

?를 사용하면 더 깔끔하게 보일 수 있을 것 같습니다!
```suggestion
  value?: number
```